src/App/App.js

```diff
@@ -0,0 +1,83 @@
+import './App.css';
+import React, { Component } from 'react';
+import Header from '../Header/Header';
+import CardContainer from '../CardContainer/CardContainer';
+import MovieDisplay from '../MovieDisplay/MovieDisplay';
+// import MovieDetails from './MovieDetails/MovieDetails';
+// import movieData from '../data/dummyData';
+import { fetchData } from '../util.js'
+
+
+class App extends Component {
+ constructor(){
+   super();
+   this.state = {
+     movies: [],
+     currentSelectionId: null,
+     error: null,
+     dummyDisplay: {
```

Comment:
I suppose once we tie our event clicks and specific movie fetch calls together, we can change the name of this key to selectedMovie and store that movie's object here. This way we won't have to majorly refactor our Movie Display component.

Reply:
Love it!